src/azul/__init__.py

```diff
@@ -364,25 +364,19 @@ def unqualified_resource_name(self,
         Traceback (most recent call last):
         ...
         azul.RequirementError: ['azul', 'object', 'versions', 'dev']
-
-        >>> config.unqualified_resource_name('azul-indexer-dev-contribute', suffix='contribute')
-        ('indexer', 'dev')
         """
-        sep = '-'
-        if suffix is not None:
-            require(len(suffix) > 0, suffix)
-            suffix = sep + suffix
-            require(qualified_resource_name.endswith(suffix))
+        require(qualified_resource_name.endswith(suffix))
+        if len(suffix) > 0:
```

Comment:
```suggestion
        if suffix:
```

Reply:
I'm making this change, but for the record, I don't understand why this is better. Also I just wanted to point out that I reverted Noah's diff exactly, so checking length was how this was done originally.

Reply:
It's better because it is shorter.